NEWS.md

```diff
@@ -1,4 +1,11 @@
-# cchsflow 0.2.1 (Latest build)
+# cchsflow 0.2.2 (Latest build)
+
+## Features:
+
+- Updated ref branch of bllflow to match latest version of `RecWTable()`
+- Minor corrections to derived variables article
```

Comment:
Add minor changes in function data and variable documentation to improve readability (or anything along those lines since those updates are in here as well)